qutip/qobj.py

```diff
@@ -1687,7 +1714,7 @@ def eigenstates(self, sparse=False, sort='low', eigvals=0,
             Use sparse Eigensolver
 
         sort : str
-            Sort eigenvalues (and vectors) 'low' to high, or 'high' to low.
+            Sort eigenvalues (and vectors) 'low' to 'high', or 'high' to 'low'.
```

Comment:
This change has removed the meaning.  I'm not saying the original was very clear, but the intent was that a value of 'low' refers to a low-to-high sort and 'high' means high-to-low.